### DISCUSSION TITLE
Installing a trained pipeline from a release in a private repo

### DISCUSSION THREAD

**Discussion by elisno** [explosion/spaCy#9119] [Help: Installation] [answered]
I've set up several spaCy projects in private repositories and figured that I could track all of the trained artifacts by trying to set up my own release repository (similar to [explosion/spacy-models](https://github.com/explosion/spacy-models), but private).

The idea is that I could do:

```bash
export OWNER=...
export RELEASE_REPO=...
export MODEL=...

pip install https://github.com/$OWNER/$RELEASE_REPO/releases/download/$MODEL/$MODEL.tar.gz
```
similarly to what is described in [the "Installation via pip" section in the docs](https://spacy.io/usage/models#download-pip)

As is, the `pip install` statement only works if the `RELEASE_REPO` is public, but I ask if anyone has any experience with doing this for a private repository?

**Comment by polm** [comment, marked as answer]
I haven't done this, but I assume you could download the `.tar.gz` with wget by specifying Github credentials somehow? If you do that then you can just `pip install` the local file.
